### DISCUSSION TITLE
10/24 반재영 블로그 생성

### DISCUSSION THREAD

**Discussion by baaanjy** [Orm-camp/blog-feedback#24] [동글동글 3기 피드백]
<a href="https://velog.io/@baaanjy/posts">https://velog.io/@baaanjy/posts</a>

저의 블로그는 벨로그에 개설했습니다.
TIL형식으로 작성하여 성장발자취를 남겨보겠습니다.
주 1회 작성 노력해보겠습니다💪

**Comment by ksj0621** [comment]
올려주신 블로그 참고해보겠습니다! 감사합니다!!

**Comment by kimjigyun** [comment]
재영님 말씀해주실때 공감 가는 포인트가 많아서 블로그 자주 보러오겠습니다 !
